caps/capabilities.go

```diff
@@ -51,9 +51,14 @@ type Repository struct {
 	caps map[string]*Capability
 }
 
+// NotFound means that a capability was not found
+type NotFound struct {
```

Comment:
For following common upstream Go conventions, can we please name this NotFoundError?  Types are FooError, variables are ErrFoo.

Also, rather than embedding an error, I suggest having a fields such as `what, name string`, so you can return &NotFoundError{"remove", name}.